frontend/src/features/mapSideBar/tabs/SectionField.tsx

```diff
@@ -0,0 +1,37 @@
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+interface ISectionFieldProps {
+  label: string;
+  content: string[];
+  isMultiLine?: boolean;
+}
+
+export const SectionField: React.FunctionComponent<ISectionFieldProps> = props => {
+  return (
+    <Row className="pb-2">
+      <Col xs="4">
+        <StyledFieldLabel>{props.label}:</StyledFieldLabel>
```

Comment:
could consider adding the : via css, that would allow easier consumers to override.